docs/architecture.md

```diff
@@ -44,11 +48,11 @@ The Composition Service consists of multiple components.
 | Service Bus | [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) responsible for reliable delivery of messages between components.  |
 | Resource Processor | Responsible for starting the process of mutating a Workspace via a Workspace Template. |
 
-The flow to provision a Workspace is the following (the flow is the same for all kinds of mutations to a Workspace):
+The flow to provision a Workspace is as follows (the flow is the same for all kinds of mutations to a Workspace):
 
-1. A HTTP request to Management API to create a new Workspace. The request contains information like the name of the Workspace, the Workspace Template to use and the parameters required for the Workspace Template (Workspace Templates can via a JSON Schema expose the the parameters).
+1. An HTTP request to Management API to create a new Workspace. The request contains information like the name of the Workspace, the Workspace Template to use, and the parameters required for the Workspace Template (Workspace Templates can expose the parameters via a JSON Schema ).
```

Comment:
```suggestion
1. A HTTP request to Management API to create a new Workspace. The request contains information like the name of the Workspace, the Workspace Template to use, and the parameters required for the Workspace Template (Workspace Templates can expose the parameters via a JSON Schema ).
```

Reply:
I believe HTTP should be an -- check out https://developer.mozilla.org/en-US/docs/Web/HTTP/Messages and https://www.ibm.com/docs/en/cics-ts/5.3?topic=protocol-http-requests